### ISSUE TITLE
html min-block-size

### ISSUE THREAD

**Issue by roblevintennis** [AustinGil/bedrocss#10]
`min-block-size` is writing mode dependent so comment [here](https://github.com/AustinGil/bedrocss/blob/master/src/base.css#L4) seems like it should be stated similar to how MDN does:
https://developer.mozilla.org/en-US/docs/Web/CSS/block-size

Here too:
https://github.com/AustinGil/bedrocss/blob/master/src/base.css#L28

**Comment by AustinGil** [comment]
Yeah, I transitioned everything over to logical properties to make it more future proof, but I think in this case I actually want to use the height property since this is intended to achieve full height content on the page.

**Comment by roblevintennis** [comment]
Yeah I think it's very likely that the mode will be:

```css
writing-mode: horizontal-tb;
```

But if:
```css
writing-mode: vertical-rl;
writing-mode: vertical-lr;
```

Then block would mean width.

I use block to mean top/bottom and inline for left/right similarly with assumption of horizontal-tb. I think simply adding `(assumes horizontal-tb writing mode)` after the height mention would suffice.

**Comment by roblevintennis** [comment]
My feedback here is just feedback no egregious bug or anything so I'll go ahead and close this and leave it up to you if anything else needs to be done. Thanks for the discussion!

**Comment by AustinGil** [comment]
Hey Rob, feel free to leave these sorts of issues open. It helps me keep track of what still needs to be addressed and I can always close it if I feel it's not something I want to pursue. In this case, I've pushed up some changes that should have fixed this issue (I hope).

**Comment by roblevintennis** [comment]
Gotcha. I'm checking from my phone and I'm not seeing a way to reopen the issue from this UI. I'll try to remember to do so once I'm at a proper computer.

**Comment by AustinGil** [comment]
Well it's been fixed now anyway ;)

**Comment by roblevintennis** [comment]
Ok, on computer -- I see https://github.com/AustinGil/bedrocss/commit/71f8c14d62f4f97bedde399c914e3d0f86743e94 which has the fix. I did leave a comment there re the `height: 100%` approach. I wonder if some folks might misunderstand that they'll only achieve that if they continually define `height: 100%` down to the element they're actually hoping to be full height ¯\_(ツ)_/¯

I think `min-height` has similar behavior:
https://developer.mozilla.org/en-US/docs/Web/CSS/min-height#formal_definition

Not sure this at all means having those in the reset in the places you put them is incorrect or anything. I just wonder if folks will understand that they will need to apply those hierarchically all the way down if that makes sense.

Reopening just so you can have a look but feel free to close you see fit ;-)
